.gitpod.yml

```diff
@@ -1,5 +1,10 @@
 tasks:
-  - init: mvn install -P quick-build
+  - init: |
+      curl -s "https://get.sdkman.io" | bash
+      echo "sdkman_auto_answer=true" > $HOME/.sdkman/etc/config
```

Comment:
Just curious, is this line L4:
```
      echo "sdkman_auto_answer=true" > $HOME/.sdkman/etc/config
```
really necessarily?

Reply:
Yes, that line is necessary because without it, it prompts you to confirm whether you want to use Java 21 as the default version. By setting sdkman_auto_answer=true, it automatically answers "yes" to this prompt, making the setup seamless without requiring manual input.